data/games_truncated.csv

```diff
@@ -0,0 +1,7 @@
+game_id,season,type,date_time,away_team_id,home_team_id,away_goals,home_goals,venue,venue_link
+2015030226,20152016,Postseason,5/11/16,15,5,3,2,Children's Mercy Park,/api/v1/venues/null
+2013030411,20132014,Postseason,6/5/14,3,26,2,3,Nippert Stadium,/api/v1/venues/null

```

Comment:
Good job on add different types of seasons

Reply:
Yeah this is great